### PR TITLE
Hardhat working on android

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-####
Figma link (if any):

## What changed (plus any additional context for devs)

- get on chain balances when "Connect to hardhat", before we were only doing it when IS_TESTING is true
- Adding localhost for android to connect to hardhat

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/components/settings-menu/DevSection.js [hunk 1]
```diff
@@ -77,17 +73,15 @@ const DevSection = () => {
     navigate(Routes.PROFILE_SCREEN);
     dispatch(explorerInit());
 
-    if (IS_TESTING === 'true') {
```
Comment (skylarbarrera) [inline src/components/settings-menu/DevSection.js:80]: smh

**Review by brunobar79 — APPROVED** [review]
🤝

**Review by terrysahaidak — APPROVED** [review]

**Comment by estrattonbailey** [comment]
@estebanmino is this fixed by the latest Android e2e work?

**Comment by terrysahaidak** [comment]
This is something we have already merged.

**Pull request closed without merging**